examples/npm_and_rest/src/gravatar.rs

```diff
@@ -38,8 +38,8 @@ impl GravatarService {
             if meta.status.is_success() {
                 callback.emit(data)
             } else {
-                // format_err! is a macro in crate `failure`
-                callback.emit(Err(format_err!(
+                // anyhow! is a macro in crate `anyhow`
```

Comment:
Feel free to remove this comment, not valuable

Reply:
Done.